reference/forms/types/options/help.rst.inc

```diff
@@ -0,0 +1,10 @@
+help
+~~~~
+
+**type**: ``string`` **default**: null
+
+Allows you to define a help text below the form field.
+
+    $builder->add('name', null, array(
+    'help'   => 'My helper message.',
+    ));
```

Comment:
Please indent the array entry with 4 spaces and you have to add `.. code-block:: php` before this code example in order to render it as PHP code.